DashAI/front/src/components/experiments/PrepareDatasetStep.jsx

```diff
@@ -0,0 +1,398 @@
+import React, { useEffect, useState } from "react";
+import PropTypes, { number } from "prop-types";
+
+import {
+  Grid,
+  TextField,
+  Typography,
+  RadioGroup,
+  FormControlLabel,
+  FormHelperText,
+  Radio,
+} from "@mui/material";
+
+/**
+ * Step of the experiment modal: Set the input and output columns to use for clasification
+ * and the splits for training, validation and testing
+ * @param {object} newExp object that contains the Experiment Modal state
+ * @param {function} setNewExp updates the Eperimento Modal state (newExp)
+ * @param {function} setNextEnabled function to enable or disable the "Next" button in the modal
+ */
+function PrepareDatasetStep({ newExp, setNewExp, setNextEnabled }) {
+  // TODO: column and row numbers should be minor to the maximum on the dataset
+  const totalColumns = 100;
+  const totalRows = 2000;
+  // columns numbers state
+  const [inputColumns, setInputColumns] = useState([]);
+  const [outputColumns, setOutputColumns] = useState([]);
+  const [columnsReady, setColumnsReady] = useState(false);
+
+  // handle columns numbers change state
+  const [parseInputColumnsError, setParseInputColumnsError] = useState(false);
+  const [parseOutputColumnsError, setParseOutputColumnsError] = useState(false);
+  const [parseInputColumnsErrorText, setParseInputColumnsErrorText] =
+    useState("");
+  const [parseOutputColumnsErrorText, setParseOutputColumnsErrorText] =
+    useState("");
+  const rangeRegex = /^(\d+)(-(\d+))*(,(\d+)(-(\d+))*)*$/;
+
+  // rows numbers state
+  const defaultParitionsIndex = {
+    training: [],
+    validation: [],
+    testing: [],
+  };
+  const defaultPartitionsPercentage = {
+    training: 0,
+    validation: 0,
+    testing: 0,
+  };
```

Comment:
Why not use the standard ones right away? For example: 60, 20, 20?

```suggestion
  const defaultPartitionsPercentage = {
    training: 60,
    validation: 20,
    testing: 20,
  };
```